README.md

```diff
@@ -0,0 +1,40 @@
+# CRC Caddy Image
+
+The CRC Caddy Image is used in the ephemeral environments to emulate the authentications used on ConsoleDot. It is deployed via the [Clowder](https://github.com/RedHatInsights/clowder) operator. The functionality is provided as a Caddy plugin.
+
+It runs in one of two modes:
+
+* **Side Car Mode** - where the configuration is taken from `/etc/Caddyfile`
+* **Gateway Mode** - where configuration is taken from `/etc/caddy/Caddyfile.json`
+
```

Comment:
```suggestion

If `/etc/caddy/Caddyfile.json` is present at startup, then the app will assume that it is running in **Gateway Mode**. Otherwise, the app will assume it should run in **Sicde Car Mode**

```